index.html

```diff
@@ -6,6 +6,392 @@
+	</section>
+	<!-- end meet Florence -->
+	<section class="row">
+		<span class="subTitle">BEHIND THE SCENES</span>
+		<h2 class="sectionDescription">While Florence is only a concept, here are some things that will be helpful in making her come alive.</h2>
+		<section id="florenceBody">
+			<figure>
+				<img src="assets/florence-capeless.svg" alt="Florence without a cape">
+				<figure class="tagLeft" id="knowledge">
+					<img src="assets/icon-knowledge-base.png" alt="A closed book with a medical cross on the cover">
+					<figcaption>
+					<h3>Medical Knowledge Bases</h3>
+					Side effects and drug interactions are currently available in services like <a href="#">Epocrates</a> and <a href="#">Medscape</a>. Though designed for long-form reading, it could be pruned and adapted to fit messaging constraints.  
+					</figcaption>`
+				</figure>
+				<figure class="tagRight" id="content">
+					<img src="assets/icon-comprehension.svg" alt="Two interlocked gears">
+					<figcaption>
+					<h3>Algorithmic Content</h3>
+					Generating believable messages that don’t feel artificial or templated is key to maintaining the illusion. Tools like <a href="#">Automated Insights Wordsmith</a> hint to possibilities where content can be produced on the fly and personalized to each user. 
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="denial">
+					<img src="assets/icon-processing.png" alt="A chat bubble">
+					<figcaption>
+					<h3>Natural Language Processing</h3>
+					Florence’s conversational style will encourage users to speak to her like a friend. Natural language processing, like those offered by <a href="#">IBM’s Watson</a>, will be needed to extract meaning and respond intelligently. 
+					</figcaption>
+				</figure>
+				<figure class="tagRight" id="cost">
+					<img src="assets/icon-denial.svg" alt="A heart symbol">
+					<figcaption>
+					<h3>Behavioral Modeling</h3>
+					Florence can generate many different messages which, unchecked, could make her a little <strong>too</strong> friendly. A system that learns the preferences of users will be helpful in tailoring which kinds of messages are sent, when, and in what frequency.
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="complexity">
+					<img src="assets/icon-integration.png" alt="A chain link">
+					<figcaption>
+					<h3>IoT Integrations</h3>
+					The emergence of connected adherence devices like <a href="#">Proteus</a> pills and <a href="#">Smrxt</a> bottles, combined with the popularity of health devices like <a href="#">Fitbits</a> and <a href="#">Qardio</a>, can provide a more rounded view of patients. Aggregating and synthesizing that information, perhaps using services like <a href="#">Validic</a>, would help Florence interact more appropriately.
+					</figcaption>
+				</figure>
+			</figure>
+		</section>
+	</section>
+	<!-- end behind the scenes -->
+
+	<!-- In Closing -->
+	<section class="row">
+		<h1>In Closing</h1>
+		<article>
```

Comment:
Yeah, this content is related to the above heading, so starting a new `<article>` is odd